frontend/src/Views/Styles/LandingPage.styles.ts

```diff
@@ -0,0 +1,111 @@
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
+import screenSizes from '../../Constants/ScreenSize';
+
+const StyledContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 265px;
+  max-height: 180px;
+  position: relative;
+  top: 15vh;
+  gap: 2px;
+  text-align: center;
+  font-family: ${tokens.typographyFontFamiliesBcSans};
+  h1 {
+    font-size: 18px;
+    font-weight: 700;
+  }
+
+  p {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 21px;
+    color: ${tokens.typographyColorSecondary};
+  }
+  @media (min-width: ${screenSizes.tablet}) {
+    top: 10vh;
+    max-width: 350px;
+
+    h1 {
+      font-size: 22px;
```

Comment:
We should use design tokens for sizing whenever possible

Reply:
sounds good!